src/julia-syntax.scm

```diff
@@ -4012,6 +4012,11 @@ f(x) = yt(x)
                (if (and tail (not have-ret?))
                    (emit-return '(null)))
                '(null)))
+
+            ;; unsupported assignment operators
+            ((≔ ⩴ ≕ :=)
+             (error (string "unsupported assignment operator \"" (deparse (car e)) "\"")))
```

Comment:
The `deparse` isn't necessary here since it's just a symbol, but it doesn't really matter.